### ISSUE TITLE
Blocked access to telemetry API cause the server to crash

### ISSUE THREAD

**Issue by kisst** [n8n-io/n8n#4905]
**Describe the bug**

Since version `0.206.1` n8n docker image  the process keep crashing, I increased the log level all the way up to `debug`, but the only lines are:

```
Error: connect ECONNREFUSED 0.0.0.0:443
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1278:16)
    at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17)
```
Rolling back to 0.206.0 works, but I have not done a bitsect to pin down the specific change which cause the trouble, just down to the release version.

**To Reproduce**

Run n8n without internet access with anything above 0.206.0 ( starting at 0.206.1)  ? 

**Expected behavior**

Process keep running, or at least the log contains a reason for the crash 

**Environment (please complete the following information):**

- OS: default docker container
- n8n Version from 0.206.0 to latest
- Database system SQLite
- Operation mode own

**Additional context**

I have n8n deployed in an internal network with no public internet access, and DNS is null-routing anything not whitelisted ( hence the 0.0.0.0 )  but could not pinpoint the domain which it might be looking for.

**Comment by d4vidsha** [comment]
I have the same issue

**Comment by kisst** [comment]
Managed to pinpoint the error and it's the telemetry API, which I disabled with 
```
version: "3"
services:
  server:
 ...
    ports:
      - 5678:5678
    environment:
      N8N_DIAGNOSTICS_ENABLED: "false"
```
as per https://docs.n8n.io/reference/data-collection/#how-collection-works

**Comment by kisst** [comment]
I updated the issue tittle too, so it's easier to spot, considering that there is a lot of home user with pi-hole and adguard and other dns level blocking systems where telemetry in the name of the domain already means by default deny ... this can be an issue for self hosted instance.

**Comment by ivov** [comment]
Closing as fixed by #4947. If it persists, please let us know and we will reopen.
